### PR TITLE
Add docs for data_compression table property in SQLServer

### PR DESCRIPTION
**Pull request by Praveen2112**

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/sqlserver.rst [hunk 1]
```diff
@@ -90,6 +90,29 @@ following aggregate functions:
 * :func:`var_pop`
 * :func:`var_samp`
 
+Table Properties
```
Comment (phd3) [inline presto-docs/src/main/sphinx/connector/sqlserver.rst:93]: @Praveen2112 thoughts providing some SQLServer reference here that explains the properties in more detail?

Comment (mosabua) [reply presto-docs/src/main/sphinx/connector/sqlserver.rst:93]: well... is any property supported? If yes.. we should list them all or link to documentation where they are listed.

File: presto-docs/src/main/sphinx/connector/sqlserver.rst [hunk 2]
```diff
@@ -90,6 +90,29 @@ following aggregate functions:
...
+      birthday DATE
+    )
+    WITH (
+      data_compression = 'ROW'
+    );
+
+==================== ================ ======================================================================================================
+Property Name        Default Value    Description
+==================== ================ ======================================================================================================
+``data_compression``   (none)           DataCompression for SQL Server tables. Can either be `NONE`, 'ROW` or `PAGE`.
```
[the remark this answers is not in the document]
Comment (mosabua) [reply presto-docs/src/main/sphinx/connector/sqlserver.rst:113]: Dont use a table if it is only one property at all. And in general might be better to use list-table (see other documentation for examples)

File: presto-docs/src/main/sphinx/connector/sqlserver.rst (hunk 1 above)
Comment (Praveen2112) [reply presto-docs/src/main/sphinx/connector/sqlserver.rst:93]: As of now we support only one property.

File: presto-docs/src/main/sphinx/connector/sqlserver.rst [hunk 3]
```diff
@@ -90,6 +90,25 @@ following aggregate functions:
 * :func:`var_pop`
 * :func:`var_samp`
 
+Table property ``data_compression``
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/sqlserver.rst:93]: Just call the section "Data Compression"

File: presto-docs/src/main/sphinx/connector/sqlserver.rst [hunk 4]
```diff
@@ -90,6 +90,25 @@ following aggregate functions:
...
+-----------------------------------
+
+With the ``data_compression`` table property you can specify the data compression
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/sqlserver.rst:96]: Turn the sentence around

You can specify the data compression policy for SQL Server tables with the ...

File: presto-docs/src/main/sphinx/connector/sqlserver.rst [hunk 5]
```diff
@@ -90,6 +90,25 @@ following aggregate functions:
...
+policy for the SQL Server tables. Date compression policy can either be
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/sqlserver.rst:97]: Valid policies are ..

Comment (mosabua) [reply presto-docs/src/main/sphinx/connector/sqlserver.rst:97]: Maybe also add a link to more info about that in the SQL Server docs somewhere..

File: docs/src/main/sphinx/connector/sqlserver.rst [hunk 6]
```diff
@@ -90,6 +90,28 @@ following aggregate functions:
 * :func:`var_pop`
 * :func:`var_samp`
 
+Table property ``data_compression``
+-----------------------------------
+
+You can specify the data compression policy for SQL Server tables with the ``data_compression``
+Valid policies are ``NONE``, ``ROW`` or ``PAGE``.
+
+See `Data Compression
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/sqlserver.rst:99]: I would get rid of this sentence and instead embed the link in the sentence above

You can specify the `data compression policy for SQL Server tables <url>`_ ....

File: docs/src/main/sphinx/connector/sqlserver.rst [hunk 7]
```diff
@@ -90,6 +90,25 @@ following aggregate functions:
 * :func:`var_pop`
 * :func:`var_samp`
 
+Table property ``data_compression``
+-----------------------------------
+
+You can specify the `data compression policy for SQL Server tables
+<https://docs.microsoft.com/en-us/sql/relational-databases/data-compression/data-compression>`_
+with the ``data_compression``. Valid policies are ``NONE``, ``ROW`` or ``PAGE``.
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/sqlserver.rst:98]:
```
with the ``data_compression`` table property.
```

File: docs/src/main/sphinx/connector/sqlserver.rst [hunk 8]
```diff
@@ -90,6 +90,25 @@ following aggregate functions:
 * :func:`var_pop`
 * :func:`var_samp`
 
+Table property ``data_compression``
+-----------------------------------
```
Comment (ssheikin) [inline docs/src/main/sphinx/connector/sqlserver.rst:94]:
```suggestion
Data Compression
----------------
```

**Review by ssheikin — APPROVED** [review]

**Review by kokosing — APPROVED** [review]

File: docs/src/main/sphinx/connector/sqlserver.rst (hunk 8 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/sqlserver.rst:94]: Data compression

(use sentence case in titles..)

Comment (Praveen2112) [reply docs/src/main/sphinx/connector/sqlserver.rst]: AC

File: docs/src/main/sphinx/connector/sqlserver.rst [hunk 9]
```diff
@@ -90,6 +90,25 @@ following aggregate functions:
 * :func:`var_pop`
 * :func:`var_samp`
 
+``Data compression``
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/sqlserver.rst:93]: Just 

Data compression

and underline should be same length as title.

**Review by mosabua — APPROVED** [review]
Only title fix remains.. then all good

**Pull request merged**